install_ffmpeg.sh

```diff
@@ -53,7 +54,52 @@ if [ ! -e "$HOME/x264" ]; then
   make install-lib-static
 fi
 
+# Static linking of gnutls on Linux/Mac
```

Comment:
We might not need to build gnutls on Mac if the ffmpeg build can be configured to use the built-in securetransport library. That should be auto-detetected under normal circumstances, but `--enable-securetransport` should work to make it explicit.

Reply:
Oh nice! I'll try running a build like that and make sure it works - I had trouble before with cert errors when I was trying to have mbedtls and gnutls talk to each other, but that could totally work yeah.

Reply:
It's quite possible that how we generate certs needs fixing as well. Last I tried, curl could not query O status, even with `--no-check-certificate`. So it'd be nice if the certs weren't *completely* bogus, but that may be a separate issue entirely.

Reply:
Hmmm, odd. I took out `--enable-gnutls` on the macOS builds. It looked to have the desired effect on `./configure`, it changed from

```
External libraries:
appkit			coreimage		zlib
avfoundation		gnutls
bzlib			libx264
```

to

```
External libraries:
appkit			coreimage		zlib
avfoundation		libx264
bzlib			securetransport
```

But then, when I try to run the binary with gnutls uninstalled...

```
▶ ./livepeer-darwin-amd64/livepeer -version                                      17:27:20
dyld: Library not loaded: /usr/local/opt/gnutls/lib/libgnutls.30.dylib
  Referenced from: /Users/iameli/code/go/src/github.com/livepeer/go-livepeer/./livepeer-darwin-amd64/livepeer
  Reason: image not found
fish: './livepeer-darwin-amd64/livepee…' terminated by signal SIGABRT (Abort)
```

Something else in the dependencies must be linking it, I suppose? Gonna try explicitly uninstalling it prior to the build.

Reply:
Ah, here's the trouble: https://github.com/livepeer/lpms/blob/master/ffmpeg/ffmpeg.go#L14

I'm sort of inclined to keep gnutls on all the platforms? - It works great and we eliminate any inconsistencies that would occur like I saw with mbedtls

Reply:
> I'm sort of inclined to keep gnutls on all the platforms?

Ah gotcha - yeah, makes sense to do that if it's most expedient.